mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -233,24 +243,92 @@ def list_all_columns(self, experiment_id: str, run_view_type: str) -> Columns:
         tags = [t.key for t in response.aggregations.tags.tags_keys.buckets]
         return Columns(metrics=metrics, params=params, tags=tags)
 
+    def _build_elasticsearch_query(self, parsed_filters: List[dict], s: Search) -> Search:
+        type_dict = {"metric": "latest_metrics", "parameter": "params", "tag": "tags"}
+        filter_key = {
+            ">": ["range", "must"],
+            ">=": ["range", "must"],
+            "=": ["term", "must"],
+            "!=": ["term", "must_not"],
+            "<=": ["range", "must"],
+            "<": ["range", "must"],
+            "LIKE": ["wildcard", "must"],
+            "ILIKE": ["wildcard", "must_not"],
```

Comment:
I'm not sure ILIKE should be must_not, ILIKE is just case insensitive like

Reply:
Oh ok I thought it was like != my bad